nixos/modules/programs/fish.nix

```diff
@@ -101,6 +101,9 @@ in
       end
     '';
 
+    # include programs that bring their own completions
+    environment.pathsToLink = [ "/share/fish/vendor_completions.d" ];
```

Comment:
When I set `programs.fish.enable = true`; and `sudo nixos-rebuild switch` on the nixos-unstable channel now, because of this line, it fails with:

```
cannot create directory `/nix/store/m3rrr54qspwiwyzn4gjrys36km6jz1g6-system-path//share/fish/vendor_completions.d': No such file or directory at /nix/store/xz5k60frmvvrj16lf7b0d8h3p0l4l7xw-builder.pl line 186.
builder for ‘/nix/store/bm5q3hs3pnijrjx1yrr647pslmzl986g-system-path.drv’ failed with exit code 2
cannot build derivation ‘/nix/store/z8bwgpzs4wiiv72rm4anc4mx1ixph1w2-nixos-system-conway-16.09pre85931.125ffff.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/z8bwgpzs4wiiv72rm4anc4mx1ixph1w2-nixos-system-conway-16.09pre85931.125ffff.drv’ failed
```

Any idea what's going on here?